test/integration/apps-test-cases.yaml

```diff
@@ -0,0 +1,263 @@
+config:
+  inherit-env: true
+
+tests:
+  01 - it successfully lists all apps:
```

Comment:
Unfortunately all tests run in alphabetical order, so we're trying to enforce an order like this.